pkg/validation/validate.go

```diff
@@ -327,6 +329,10 @@ type ValidatedRangeRequest struct {
 }
 
 func ValidateRangeRequest(limits RangeRequestLimits, tenantIDs []string, req model.Interval, now model.Time) (ValidatedRangeRequest, error) {
+	if req.Start == 0 && req.End == 0 {
+		return ValidatedRangeRequest{}, NewErrorf(QueryMissingTimeRange, QueryMissingTimeRangeErrorMsg)
+	}
```

Comment:
Should we require both `Start` and `End`?
```suggestion
	if req.Start == 0 || req.End == 0 {
		return ValidatedRangeRequest{}, NewErrorf(QueryMissingTimeRange, QueryMissingTimeRangeErrorMsg)
	}
```

Reply:
Yeah, you are right. I think this is a good suggestion 👍